pycbc/workflow/plotting.py

```diff
@@ -45,6 +45,10 @@ class PlotExecutable(Executable):
     """ plot executable
     """
     current_retention_level = Executable.FINAL_RESULT
+    
+    # plots and final results should get the highest priority
+    # on the job queue
+    self.set_priority(100)
```

Comment:
Change this to 1000.

I think priority seems to be set (automatically) in intervals of 10 according to the depth of a DAX. So a dax with 10 levels might actually start setting priorities to 100.